api/src/Controller/Announcement/ListDepartmentAnnouncements.php

```diff
@@ -3,6 +3,63 @@
     require_module 'standard';
 .*/
 
+/**
+ *  @OA\Get(
+ *      tags={"departments"},
+ *      path="/department/{id}/announcements",
+ *      summary="Lists announcements for a given department",
+ *      @OA\Parameter(
+ *          description="The id or name of the department",
+ *          in="path",
+ *          name="id",
+ *          required=true,
+ *          @OA\Schema(
+ *              oneOf = {
+ *                  @OA\Schema(
+ *                      description="Department name",
+ *                      type="integer"
+ *                  ),
+ *                  @OA\Schema(
+ *                      description="Department id",
+ *                      type="string"
+ *                  )
+ *              }
+ *          )
+ *      ),
+ *      @OA\Parameter(
+ *          description="Include the resource instead of the ID.",
+ *          in="query",
+ *          name="include",
+ *          required=false,
+ *          explode=false,
+ *          style="form",
+ *          @OA\Schema(
+ *              type="array",
+ *              @OA\Items(
+ *                  type="string",
+ *                  enum={"departmentId","postedBy"}
```

Comment:
`department` rather than `departmentId`, but that can also wait until we do the shift to snake_case, since we discussed it all in a bundle.